packages/vats/src/localchain.js

```diff
@@ -17,32 +48,34 @@ const prepareLocalChainAccount = zone =>
     'LocalChainAccount',
     LocalChainAccountI,
     /**
-     * @param {import('./types.js').ScopedBridgeManager} system
      * @param {string} address
-     * @param {{ query: (messages: any[]) => Promise<any> }} chain
+     * @param {PowerStore} powers
      */
-    (system, address, chain) => ({ system, address, chain }),
+    (address, powers) => ({ address, powers }),
     {
       // Information that the account creator needs.
       async getAddress() {
         return this.state.address;
       },
-      async allBalances() {
-        // We make a balance request, scoped to our own address.
-        const { address, chain } = this.state;
-        const res = await chain.query([
-          {
-            '@type': '/cosmos.bank.v1beta1.QueryAllBalancesRequest',
-            address,
-          },
-        ]);
-        if (res[0].error) {
-          throw Fail`query failed: ${res[0].error}`;
-        }
-        return res[0].reply;
+      /**
+       * Deposit a payment into the bank purse that matches the alleged brand.
+       * This is safe, since even if the payment lies about its brand, ERTP will
+       * reject spoofed payment objects when depositing into a purse.
+       *
```

Comment:
I don't expect this change for this PR, but I think we should have a solid story around ERTP payment recovery sets and how to use them to prevent orphaned payments if the `deposit` or `getAllegedBrand` methods get stuck.